src/templates/node/index.ts

```diff
@@ -29,14 +28,27 @@ Pond.default().then((pond) => {
 })
 `
 
-export const dockerfile = (appName: string): string => `FROM node:10-alpine
+export const dockerfile = (appName: string): string => `FROM node:10-alpine as build
 WORKDIR /usr/src/app
 
-COPY package*.json ./
+COPY src/${appName}/package-prod.json ./package.json
 RUN npm install --production
 COPY build/${appName}/. .
 
-CMD ["node", "index.js"]
+FROM node:10-alpine
+COPY --from=build /usr/src/app /
+CMD ["node", "${appName}/index.js"]
+`
+
+export const packageJsonProd = (appName: string): string => `{
+  "name": "${appName}",
+  "version": "1.0.0",
```

Comment:
should the version pass as a parameter taken from package.json?

Reply:
very good idea. but this is a bit tricky. The file is generated with `axp add node` and is then fix.

It is not updating during the build.